gp-address-autocomplete/gpaa-single-line-as-input.php

```diff
@@ -107,5 +113,6 @@ public function is_applicable_form( $form ) {
 new GPAA_Single_Line_Input( array(
 	'form_id'              => 123,     // The ID of your form.
 	'address_field_id'     => 4,       // The ID of the Address field.
-	'single_line_field_id' => 5,        // The ID of the Single Line Text field.
+	'single_line_field_id' => 5,       // The ID of the Single Line Text field.
+	'use_full_address'     => true,    // Whether to use full street address or not 
```

Comment:
```suggestion
	'single_line_field_id' => 5,       // The ID of the Single Line Text field.
	// 'use_full_address'     => true,    // Uncomment to use the full street address if you don't want an abbreviated street address.
```

(it's hard to get whitespace right in this suggestion 😆)